src/Managers/FileManager.php

```diff
@@ -73,8 +73,8 @@ private function getPhpFilesFromPath(string $path)
     private function fileShouldBeIgnored(SplFileInfo $file): bool
     {
         foreach ($this->filesToIgnore as $fileToIgnore) {
-            $fileToIgnore = str_replace('/', '\/', $fileToIgnore);
-            if (preg_match("/{$fileToIgnore}/", $file->getPathName())) {
+            $fileToIgnore = preg_replace("#/(.*)\*$#", "/$1.+$", $fileToIgnore);
```

Comment:
Could you extract this to its own method? It's not obvious to me what this line does. Having it extracted to its own method with a doc block will make it much more clear I think.